src/Sulu/Component/DocumentManager/Slugifier/PathCleanupSlugifier.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\DocumentManager\Slugifier;
+
+use Sulu\Component\PHPCR\PathCleanupInterface;
+use Symfony\Cmf\Api\Slugifier\SlugifierInterface;
+
+class PathCleanupSlugifier implements SlugifierInterface
+{
+    /** @var PathCleanupInterface */
+    private $pathCleanup;
+
+    public function __construct(PathCleanupInterface $pathCleanup)
+    {
+        $this->pathCleanup = $pathCleanup;
+    }
+
+    public function slugify($text)
+    {
+        $text = $this->pathCleanup->cleanup($text);
+        $text = str_replace('/', '-', $text);
+
+        // Remove apostrophes which are not used as quotes around a string
+        $text = preg_replace('/(\\w)\'(\\w)/', '${1}${2}', $text);
```

Comment:
is there a specific reason for applying exactly these filters here? 🙂

Reply:
I copied the old ones, hoping they will fix the tests. :) I don't have PDO installed on this machine and I cannot run functional tests. :(